lib/Sema/CSGen.cpp

```diff
@@ -1750,6 +1750,24 @@ namespace {
           return nullptr;
         }
 
+        if (numElements == 1 && CS.isForCodeCompletion()) {
```

Comment:
Actually more one thing - maybe we should checking - if there is one element and it's a code completion expression?